documentation/src/mkdocs.yml

```diff
@@ -50,6 +83,17 @@ pages:
       - 'Java API Reference': 'javadoc.md'
       - 'REST API - Controller': 'rest/restapis.md'
       - 'Pravega Connectors': 'connectors.md'
+      - 'Flink Connectors':
```

Comment:
Should we add Schema Registry in here as well?

Reply:
I've asked @shiveshr if he can find a way to update existing MkDocs with Schema Registry information.